src/app/chatbot-page/interview/interview.component.scss

```diff
@@ -1,249 +1,225 @@
 @import url('https://fonts.googleapis.com/css2?family=Rubik:ital,wght@0,300..900;1,300..900&display=swap');
+
 * {
-    box-sizing: border-box;
+  box-sizing: border-box;
 }
 
 .interview {
-    display: flex;
-    justify-content: space-evenly;
-    gap: 20px;
-
-
-  
-    
+  display: flex;
+  justify-content: space-evenly;
+  gap: 20px;
 }
 
 @media (max-width: 768px) {
-    .interview {
-        flex-direction: column;
-        max-width: none;
-        width: 100%;
-    }
-
-
-
+  .interview {
+    flex-direction: column;
+    max-width: none;
+    width: 100%;
+  }
 }
 
-
-
 .questions {
-    display: flex;
-    -webkit-box-pack: start;
-    justify-content: flex-start;
-    -webkit-box-align: center;
-    align-items: center;
-    flex-direction: column;
-    width: 420px;
-    margin-right: 92px;
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  flex-direction: column;
+  width: 420px;
+  margin-right: 92px;
 }
 
 .interview .questions div h2 {
-    font-size: 50px;
-    animation: text-animation 1s ease-in-out;
-    overflow: auto;
+  font-size: 50px;
+  animation: text-animation 1s ease-in-out;
+  overflow: auto;
 }
+
 @media screen and (max-width: 768px) {
-    .interview .questions div h2 {
-        font-size: 30px;
-    }
-    
+  .interview .questions div h2 {
+    font-size: 30px;
+  }
 }
 
 @keyframes text-animation {
-    0% {
-        opacity: 0;
-        transform: translateY(20px);
-    }
-    100% {
-        opacity: 1;
-        transform: translateY(0);
-    }
+  0% {
+    opacity: 0;
+    transform: translateY(20px);
+  }
+  100% {
+    opacity: 1;
+    transform: translateY(0);
+  }
 }
 
 .answers {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    max-width: 50%;
-    width: 50%;
-    min-height: 100%;
-
- 
-
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  max-width: 50%;
+  width: 50%;
+  min-height: 100%;
 }
 
 .answers div {
-    width: 100%;
-}
-.MultipleChoiceAnswers {
-    display: flex;
-    flex-wrap: wrap;
-    gap: 10px;
-    justify-content: left;
-    
+  width: 100%;
+
 }
 
+.MultipleChoiceAnswers {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+  justify-content: left;
 
+}
 
-mat-button-toggle-group { //Rahmen in dem alle Antwort buttons sind;
-    flex-wrap: wrap;
- 
-    gap: 10px;
-    outline: none;
-    padding: 20px; /* Fügt innen Abstand hinzu und vergrößert somit den Rahmen */
-    margin: 20px;  /* Fügt außen Abstand hinzu und vergrößert den Abstand um den Rahmen herum */
-    box-shadow: rgba(0, 0, 0, 0.07) 0px 0px 25px 5px;
-    border-radius: 30px; /* Fügt abgerundete Ecken hinzu */
-    overflow-y: auto;
+mat-button-toggle-group {
+  flex-wrap: wrap;
+  gap: 10px;
+  outline: none;
+  padding: 20px;
+  margin: 20px;
+  box-shadow: rgba(0, 0, 0, 0.07) 0px 0px 25px 5px;
+  border-radius: 30px;
+  overflow-y: auto;
 }
-//nur für die Auswahl ,,Englisch"
```

Comment:
I would translate this comment to english and keep it there in case we need to do a quick change to the buttons later